pages/styles/index.css

```diff
@@ -283,21 +299,49 @@ main {
   background-size: 3vw;
 }
 
+/******** Sponsors *********/
+
+#sponsors {
+  display: none;
+}
+
+/**** Footer ****/
+
+footer {
+  background-color: var(--drkGray);
+}
+
+.footer-p {
+  color: var(--lghtGray);
+}
+
+footer p a {
+  color: var(--lghtGray);
+}
+
+/*************************************** Desktop *********************************************/
+
 @media screen and (min-width: 768px) {
```

Comment:
Good job on the design 🎨 but unfortunately you're facing some overflow issues at some screen sizes, the design is not taking the whole width of the screen and the overflow is showing. See below 👇🏽 

![Screenshot 2023-02-10 at 21 39 42](https://user-images.githubusercontent.com/45739963/218234803-3f927eb4-9631-48fe-8eb6-d698787de7ec.png)
![Screenshot 2023-02-10 at 21 40 03](https://user-images.githubusercontent.com/45739963/218234824-ef4b4dc1-1bc7-43fc-8109-837df2980dc3.png)

NB: The same problem is also in the about section. 😉